### PR TITLE
Remove useless tests in platform

### PR DESCRIPTION
**Pull request by ahocquard**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/legacy/features/platform/acl/assert_acl.feature [hunk 1]
```diff
@@ -65,39 +46,4 @@ Feature: Define user rights
...
-      | Remove an association type  | "X_SELL" association type                    | Delete |
-      | Remove an attribute group   | "Sizes" attribute group                      | Delete |
-      | Remove a category           | "sandals" category                           | Delete |
-      | Remove a family             | "boots" family                               | Delete |
-      | Remove a group              | "similar_boots" product group                | Delete |
-      | Remove a group type         | "RELATED" group type                         | Delete |
       | Remove a product            | "boot" product                               | Delete |
-      | Remove an attribute         | "color" attribute                            | Delete |
-      | Remove an export profile    | "csv_footwear_option_export" export job edit | Delete |
-      | Remove an import profile    | "csv_footwear_group_import" import job edit  | Delete |
```
Comment (jmleroux) [inline tests/legacy/features/platform/acl/assert_acl.feature]: Same here.
The technical part may work by testing just "remove a product", but we don't test that other screens are well configured.

File: tests/legacy/features/platform/export/edit_an_export.feature [hunk 2]
```diff
@@ -61,26 +51,3 @@ Feature: Edit an export
...
     And I should see the text "Not complete on all selected locales"
     And I should see the text "identifier1, identifier2, identifier3, identifier4"
-
-  @skip-nav
-  Scenario: Successfully display a dialog when we quit a page with unsaved changes
-    Given I am on the "csv_footwear_product_export" export job edit page
-    When I fill in the following information:
-      | Label | My export |
-    When I click on the Akeneo logo
-    Then I should see "You will lose changes to the export profile if you leave this page." in popup
```
Comment (jmleroux) [inline tests/legacy/features/platform/export/edit_an_export.feature]: Is it an edge case?
